src/test/kotlin/com/vk/tlschema/highlighting/TLSchemaSyntaxHighlighterTest.kt

```diff
@@ -0,0 +1,5 @@
+package com.vk.tlschema.highlighting
+
+class TLSchemaSyntaxHighlighterTest : TLSchemaSyntaxHighlighterTestBase() {
```

Comment:
I think that u should rename this class to `TLSchemaSyntaxHighlightingTest` too, because highlighter is `noun`